Untitled.py

```diff
@@ -0,0 +1,53 @@
+
+
+
+def sheet_data(sheet):
+
+    out = list()
+
+    sheetIndex = 0
+    for X in range(1, s.nrows):
+        learner = s.row_values(X)
+        print(learner)
+        average=sum(learner)
+        out.append([learner[1], average])
+
```

Comment:
No blank line before return is required

Reply:
Empty lines can improve readability and should not be treated as an error.